e2etest/l2tests/assignment.go

```diff
@@ -200,13 +200,24 @@ var _ = ginkgo.Describe("IP Assignment", func() {
 					AllocateTo: &metallbv1beta1.ServiceAllocation{Priority: 20, Namespaces: []string{f.Namespace.Name}},
 				},
 			}
+			testNs, err := cs.CoreV1().Namespaces().Get(context.Background(), f.Namespace.Name, metav1.GetOptions{})
```

Comment:
I'd just add a couple of fixed labels to the current one (like foo=aaa, bar=bbb), update the service and use those fixed that we control as selectors.

Reply:
done, I hope that's what you intended to do.

Reply:
yep, this will allow us to have more control assuming nothing will modify our labels